### PR TITLE
Added a boundary event test for multi-pointers.

### LINKED ISSUE #3156: Need tests for compat mouse events for multiple pointers
A recent [fix](https://github.com/w3c/pointerevents/pull/56) in the PointerEvent spec calls for new tests with compatibility mouse events. The spec now clarifies that with multiple active pointers, the compat mouse boundary events behave as if there's a single compat mouse that "follow" each active pointer.

### PR DESCRIPTION
**Pull request by mustaqahmed**
Also refacotored event log into the support js.

Closes #3156

### CODE REVIEW

File: pointerevents/pointerevent_multiple_primary_pointers_boundary_events-manual.html [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+  </head>
+  <body onload="run()">
+    <h1>Pointer Event: Boundary compatibility events for multiple primary pointers</h1>
+    <h4>
+      When more than one primary pointers are active, each will have an independent sequence of pointer boundary events but the compatibilty mouse boundary events have their own sequence.
+    </h4>
+    Instruction:
+    <ol>
+      <li>Move the mouse directly into Target0 (without going through Target1), and then leave the mouse there unmoved.</li>
+      <li>Tap directly on Target1 with a finger or a stylus, and then move the finger/stylus off the screen/digitizer without crossing Target1 boundary.</li>
```
Comment (RByers) [inline pointerevents/pointerevent_multiple_primary_pointers_boundary_events-manual.html]: nit:
by "move the finger/stylus off" do you mean "lift the finger/stylus off"?

**Comment by RByers** [comment]
Just one little wording nit, otherwise seems good to me.

File: pointerevents/pointerevent_multiple_primary_pointers_boundary_events-manual.html (hunk 1 above)
Comment (mustaqahmed) [inline pointerevents/pointerevent_multiple_primary_pointers_boundary_events-manual.html]: Done.

**Comment by RByers** [comment]
LGTM, thanks.

**Pull request merged**